typescript/infra/scripts/check/check-warp-deploy.ts

```diff
@@ -13,7 +13,7 @@ import {
 } from './check-utils.js';
 
 async function main() {
-  const { environment, asDeployer, chain, fork, context, pushMetrics } =
+  const { environment, asDeployer, chains, fork, context, pushMetrics } =
```

Comment:
Would you mind running this script to check that it works, I should add a CI test for it

Reply:
```sh
$ yarn tsx scripts/check/check-warp-deploy.ts -e mainnet3 -c ethereum
bigint: Failed to load bindings, pure JS will be used (try npm run rebuild?)

Checking warp route USDC/ancient8-ethereum...
warp checker found no violations

Checking warp route EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-zircuit...
warp checker found no violations

Checking warp route ECLIP/arbitrum-neutron...
warp checker found no violations

Checking warp route TIA/arbitrum-neutron...
warp checker found no violations

Checking warp route USDC/ethereum-inevm...
warp checker found no violations

Checking warp route USDT/ethereum-inevm...
warp checker found no violations

Checking warp route ETH/ethereum-viction...
warp checker found no violations

Checking warp route USDC/ethereum-viction...
warp checker found no violations

Checking warp route USDT/ethereum-viction...
warp checker found no violations

Checking warp route INJ/inevm-injective...
warp checker found no violations

Checking warp route TIA/mantapacific-neutron...
warp checker found no violations
```

Reply:
is there any other specific thing to check here?